.gitignore

```diff
@@ -66,6 +66,7 @@ coverage.xml
 /test/units/cover-html
 /test/integration/targets/*/backup/
 /test/cache/*
+metadata-*.json
```

Comment:
This change should be made in a separate PR since it's not directly related to the rest of the changes here.